e2eTest/project_test.go

```diff
@@ -314,6 +314,51 @@ func TestProjects(t *testing.T) {
 
 }
 
+func TestProjectSavedTime(t *testing.T) {
+	c := newClient()
+
+	var projResp1 CreateProjectResponse
+	err := c.Post(
+		MutationCreateProject,
+		&projResp1,
+		client.Var("title", "foo1"),
+		client.Var("description", "bar"),
+		client.Var("readme", "bah"),
+		client.Var("seed", 42),
+		client.Var("numberOfAccounts", initAccounts),
+	)
+	require.NoError(t, err)
+	require.NotEmpty(t, projResp1.CreateProject.UpdatedAt)
+
+	cookie := c.SessionCookie()
+	projectID := projResp1.CreateProject.ID
+
+	var projResp2 GetProjectResponse
+	err = c.Post(
+		QueryGetProject,
+		&projResp2,
+		client.Var("projectId", projectID),
+	)
+	require.NoError(t, err)
+	require.NotEmpty(t, projResp2.Project.UpdatedAt)
```

Comment:
If you just want to check presence of this field you can just add it in the existing test to create a project. I think it would be useful to at least assert the second date is later than the first. Also that it's withing maybe few seconds of current time.

Reply:
Updated test to check time difference